starboard/build/config/modular/BUILD.gn

```diff
@@ -208,3 +208,20 @@ config("no_pedantic_warnings") {
     "-Wno-undefined-var-template",
   ]
 }
+
+# These are specifically for modular builds of Cobalt shared libraries.
+# They wrap underlying POSIX implementations to be cross-compatible with
+# musl-based types that are used in modular builds. Evergreen builds use
+# a different map-based wrapping of these POSIX APIs.
+# Note that the corresponding implementations of these wrappers need to be
+# compiled into the Starboard shared library, which is handled in
+# //starboard/build/config/starboard_target_type.gni.
+config("posix_wrappers_for_shared_libraries") {
+  if (sb_is_modular && !sb_is_evergreen &&
+      current_toolchain != starboard_toolchain) {
```

Comment:
I assume you only want this when `current_toolchain == cobalt_toolchain`, as the current conditional implies it's also true for the host toolchain (even though this config wouldn't get applied)